modules/govuk/manifests/nodes/s_postgresql_db_admin_base.pp

```diff
@@ -0,0 +1,79 @@
+# == Class: govuk::nodes::s_postgresql_db_admin_base
+#
+# This machine class defines the common configuration used to
+# administer the app-specific DB-admin machines for PostgreSQL
+# RDS instances.
+#
+# === Parameters
+#
+# [*postgres_host*]
+#   Hostname of the RDS database to use.
+#   Default: undef
+#
+# [*postgres_user*]
+#   The PostgreSQL user to use for admisistering the database.
+#   Default: undef
+#
+# [*postgres_password*]
+#   The password corresponding to the above `postgres_user`.
+#   Default: undef
+#
+# [*postgres_port*]
+#   The port with which to connect to the `postgres_host`.
+#   Default: '5432'
+#
+class govuk::nodes::s_postgresql_db_admin_base(
```

Comment:
```suggestion
class govuk::nodes::postgresql_db_admin(
```

Since this isn't an actual node I think we can drop the weird `s_` (not too sure what that is for) and I think removing `_base` makes since since we don't expect this to be an inherited class any more.